utopia-vscode-common/src/mailbox.ts

```diff
@@ -70,27 +70,44 @@ async function initOutbox(outboxToUse: Mailbox): Promise<void> {
   }
 }
 
-async function receiveMessage<T>(messageName: string, parseMessage: (msg: string) => T): Promise<T> {
+async function receiveMessage<T>(
+  messageName: string,
+  parseMessage: (msg: string) => T,
+): Promise<T> {
   const messagePath = pathForInboxMessage(messageName)
   const content = await readFileSavedContentAsUTF8(messagePath)
   return parseMessage(content)
 }
 
+async function waitForPathToExist(path: string, maxWaitTime: number = 5000): Promise<void> {
```

Comment:
this function kind of makes me uneasy because it has no explicit return at the end, so the happy path (`Promise.resolve()`) is totally implicit.

Reply:
*nods in agreement*

Reply:
Fixed in e7a5df1c.